rmw_fastrtps_dynamic_cpp/src/rmw_event.cpp

```diff
@@ -61,4 +61,10 @@ rmw_event_set_callback(
     callback,
     user_data);
 }
+
+bool rmw_event_check_compatible(rmw_event_type_t rmw_event_type)
```

Comment:
I guess we need something similar in `rmw_fastrtps_cpp/src/rmw_event.cpp`

Reply:
I can't see any `set` or structure in `rmw_fastrtps_cpp` with the supported `events`.

Reply:
What I mean is that you only added the new rmw function in one of the two RMW implementations on this repository.

So you should also add the same lines [here](https://github.com/ros2/rmw_fastrtps/blob/78452b869b27ed57651fb05f8b55049cad7b6c26/rmw_fastrtps_cpp/src/rmw_event.cpp#L63-L64)

Reply:
the internal function `is_event_supported` is define in `rmw_fastrtps_shared_cpp/src/types/event_types.hpp` which is not public at this moment.

Reply:
I see. So additionally to my suggestion, we need to export a function in `rmw_fastrtps_shared_cpp`, similar to what we do [here](https://github.com/ros2/rmw_fastrtps/blob/78452b869b27ed57651fb05f8b55049cad7b6c26/rmw_fastrtps_cpp/src/rmw_publisher.cpp#L146)